homework-submission/react-hyf/react-hyf-client/src/Components/Homework3/Homework3.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import './Homework3.css';
+import { getUsers } from './services/usersServices';
+import UserList from './Components/UserList/UserList';
+
+/**
+ * This class component is the container of the Users List
+ * @return Here is the return of the Users List
+ */
+class Homework3 extends Component {
+  state = {};
```

Comment:
in a strict mode this will cause an error, which is something you may encounter.
what about initializing the users array here before fetching?

Reply:
Many thanks here also for this advice... Done!